pkg/cli/debug.go

```diff
@@ -25,17 +27,18 @@ func newDebugCommand() *cobra.Command {
 	}
 
 	cmd.AddCommand(debug)
+	cmd.Flags().StringVarP(&imageName, "image-name", "", "", "The image name to use for the generated Dockerfile")
 
 	return cmd
 }
 
 func cmdDockerfile(cmd *cobra.Command, args []string) error {
-	config, projectDir, err := config.GetConfig(projectDirFlag)
```

Comment:
Huh, I guess we were shadowing this before. I wonder if this was being caught by linters and not being surfaced in an actionable way...